data/yaml/venues/unlp.yaml

```diff
@@ -0,0 +1,2 @@
+acronym: UNLP
+name: Workshop on Ukrainian Natural Language Processing (UNLP 2023)
```

Comment:
In the venue entry, there shouldn't be a year

Reply:
Also no acronym in the name (we show that from the acronym field already)